packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -65,7 +72,7 @@ describe('HasMany relation', () => {
   //--- HELPERS ---//
 
   @model()
-  class Customer extends Entity {
+  class Order extends Entity {
```

Comment:
Hmm, is it required for `Order` to be defined before `Customer`? 

If it is, then how are we going to support the case where `Order` has a property `customer` for "belongsTo" relation and `Customer` has a property `orders` for "hasMany" relation? 

If the order is not relevant, then please revert this part of the change and keep `Customer` model defined first, `Order` model second.

Reply:
Since `orders` property of Customer needs its type to be defined as `Order[]`, Order needs to come first. What you bring up is pretty concerning though since we may end up in a dependency cycle this way with `belongsTo` (e.g. `Order`'s `customerId` requiring a modelFrom (`Customer`), and `Customer`'s `orders` requiring modelTo (`Order`)).

I'm not sure what the best solution to this problem is nor am I aware of which one can be left out :(

Reply:
Just figured out that we don't need a `modelTo` key, meaning the dependency cycle should now be resolved 🎉